tests/behat/features/cu_advanced_design_bundle/context/context.feature

```diff
@@ -1,8 +1,11 @@
 Feature: Context
+In order to place blocks and add background graphics
+As an authenticated user
+I should be able to access the full context features
 
   @context @contextconditions @exclude_all_bundles
-  Scenario Outline: A content_editor should see a limited number of context conditions
-    Given  I am logged in as a user with the "content_editor" role
+  Scenario Outline: A Site Owner should see a limited number of context conditions
```

Comment:
Do other roles need to be tested? Each time multiple roles are tested it adds to the test time, but if it is important enough to not have the conditions and reactions available, then it might be worth a shot. Also, it looks like this is not a JS test so it's a lot faster.